console-subscriber/src/lib.rs

```diff
@@ -38,6 +38,11 @@ pub use builder::{init, spawn};
 use crate::aggregator::Id;
 use crate::visitors::{PollOpVisitor, StateUpdateVisitor};
 
+/// A [`ConsoleLayer`] is a [`tracing_subscriber::Layer`] that catches the tracing
+/// events of the runtime (e.g. call-sites), and translating them into more
+/// abstract events (e.g. task spawns or waker operations) that are meaningful
+/// to modeling the async runtime, and then transmitting those abstact events
+/// to an associated [`Server`] for aggregation.
```

Comment:
this is one _very_ long sentence, can we split it up? here's a suggestion:

```suggestion
/// A [`ConsoleLayer`] is a [`tracing_subscriber::Layer`] that records [`tracing`]
/// spans and events emitted by the async runtime. 
///
/// Runtimes emit [`tracing`] spans and events that represent specific operations
/// that occur in asynchronous Rust programs, such as spawning tasks and waker
/// operations. The `ConsoleLayer` collects and aggregates these events, and the
/// resulting diagnostic data is exported to clients by the corresponding gRPC 
/// [`Server`] instance.
///
/// [`tracing`]: https://docs.rs/tracing
```